lib/synced/model.rb

```diff
@@ -36,26 +36,23 @@ module Model
     #   on synchronized object and delegated to synced_data Hash
     # @option options [Hash] query_params: Given attributes and their values
     #   which will be passed to api client to perform search
-    def synced(options = {})
-      options.symbolize_keys!
+    def synced(strategy: :updated_since, **options)
```

Comment:
Add changelog info to force `full` strategy in models that do not contain `synced_all_at` column.